app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class UserMailer < ActionMailer::Base
+  default from: "no-reply@bpac.com"
+  layout 'mailer'
+
+  def invite(user_id)
```

Comment:
done - locally. I change the name to "invitation" because then the code reads "invitation().deliver_now" which seems elegant and not redundant.